src/task/task.entity.ts

```diff
@@ -1,4 +1,25 @@
-import { Entity } from "typeorm";
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from "typeorm";
+
+import { User } from "src/user/user.entity";
 
 @Entity()
-export class TaskEntity { }
\ No newline at end of file
+export class Task {
+
+	@PrimaryGeneratedColumn()
+	public id: number;
+
+	@Column()
+	public title: string;
+
+	@Column()
+	public description: string;
+
+	@Column()
+	public is_completed: boolean;

```

Comment:
```suggestion
	@Column({type: "text", name: "is_completed", nullable: false})
	public isCompleted: boolean;

```
refactor for all the properties